packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DateComponents/TodayButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TodayButton extends React.Component {
+  setToday = () => {
+    const { setSelected } = this.props;
+    if (setSelected) setSelected(new Date());
+  };
+  render() {
+    return (
+      <table className="table-condensed">
+        <tbody>
+          <tr>
+            <td>
+              <button className="today-button" onClick={this.setToday} title="Go to today">
```

Comment:
please add ` type="button"`, without a type in forms, clicking on this  would try to submit

Reply:
Added, thanks